apps/i18n/common/en_us.json

```diff
@@ -497,7 +497,6 @@
   "noClassroomsFound": "No classrooms found.",
   "noIconsFound": "No icons found",
   "noMenuItemsAvailable": "No menu items available.",
-  "none": "None",
```

Comment:
it is not safe to remove this because it is also used in apps/src/templates/census2017/censusQuestions.js